### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by JesusMtnez**
This PR update multiple versions of libraries and tools:

- **Scala**
  - `2.11.11` -> `2.11.12`
  - `2.12.4`  -> `2.12.8`
- **sbt**: `1.1.0` -> `1.2.8`
  - **sbt-release**: `1.0.7` -> `1.0.11`
  - **sbt-pgp**: `1.1.0` -> `1.1.2`
  - **sbt-coveralls**: `1.2.2` -> `1.2.7`
  - **tut-plugin**: `0.6.2` -> `0.6.10`
- **nscala-time**: `2.18.0` -> `2.22.0`
- **htmlunit** [1]: `2.29` -> `2.34.1`
- **jsoup**: `1.11.2` -> `1.11.3`
- **scalaz-core**: `7.2.18` -> `7.2.27`
- **specs2-core**: `4.0.2` -> `4.5.1`
- **config**: `1.3.2` -> `1.3.3`

Updating `htmlunit` required a minor change in the test since `BrowserVersion.FIREFOX_45` has been removed.

`http4s` has not been updated since it requires a refactor in test because its API has change from `0.17.x` to `0.18.x`. IMHO, it would a better idea to address that change in another PR.

### CODE REVIEW

**Comment by JesusMtnez** [comment]
@ruippeixotog sorry to ping you, but is this project active?

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ sudo: false
 
 language: scala
 scala:
-  - 2.11.11
-  - 2.12.4
+  - 2.11.12
+  - 2.12.7
```
Comment (ruippeixotog) [inline .travis.yml]: Can you update to version 2.12.8 here?

**Comment by JesusMtnez** [comment]
If you want, I could help you maintain this library. Let me know what you think.

File: .travis.yml (hunk 1 above)
Comment (JesusMtnez) [reply .travis.yml]: Done!

**Comment by ruippeixotog** [comment]
@JesusMtnez do you have ideas for new features, or do you mean bumping dependencies and publishing new versions regularly? Regarding the latter I believe I'll be able to dedicate more time to this and automate the process soon, but new ideas are always welcome :)

**Comment by JesusMtnez** [comment]
I can help you with both. At this moment I do not have any new features on my mind, but I can help with any new features you may have in mind.

Also, I could help keeping this project up to date and improving CI if needed.

**Pull request merged**